Troubleshooting Node profiler issue (#33075)
DataDog/documentation: content/en/profiler/profiler_troubleshooting/nodejs.md

When the Datadog profiler is enabled, it periodically sends a small signal to collect profiling data. If the application is idle (waiting for I/O or timers), this signal briefly wakes the event loop to take a sample, even though there's no real work to do. As a result, ELU appears higher because the loop wakes up more often, but CPU usage stays low, and no latency or extra workload is introduced.
In a busy application, these same signals occur while the loop is already active, so the effect on ELU is negligible.

## Disabling Advanced Profiling Features

Advanced profiling capabilities in Datadog provide rich context—such as endpoint-level visibility, timeline correlations, and code hotspot linking—that significantly improve analysis quality.
Under normal circumstances, these should remain enabled. However, when diagnosing performance issues or overhead, temporarily disabling specific components can help isolate the source of additional load.

> The goal is to narrow down which feature contributes to CPU, memory, or latency changes during profiling.

Reviewer: ```suggestion
You can do so to narrow down which feature contributes to CPU, memory, or latency changes during profiling.
```

 
1. Disable Code Hotspots
   
`DD_PROFILING_CODEHOTSPOTS_ENABLED=false` Disables trace-to-profile linking in Datadog UI.

2. Disable Timeline View

`DD_PROFILING_TIMELINE_ENABLED=false` Disables timeline view in the Datadog UI.

3. Disable Endpoint Collection

`DD_PROFILING_ENDPOINT_COLLECTION_ENABLED=false` Disables per-endpoint aggregation of profiling data, this prevents grouping samples by HTTP endpoints or routes

> Once you identify that disabling a specific profiling feature reduces overhead, please report your findings so we can investigate further. You can contact [Datadog support][5]

Reviewer: ```suggestion
After you identify that disabling a specific profiling feature reduces overhead, report your findings so Datadog can investigate further. You can do so by contacting [Datadog support][5].
```
